server/chat/backend/agent/prompt/prompt_builder.py

```
@@ -1159,10 +1159,32 @@ def build_system_invariant() -> str:
-def build_regional_rules() -> str:
+def _get_user_aws_default_region(user_id: Optional[str]) -> Optional[str]:
+    """Look up the default region from the user's primary AWS connection."""
+    if not user_id:
+        return None
+    try:
+        from utils.db.connection_utils import get_user_aws_connection
+        conn = get_user_aws_connection(user_id)
+        if conn:
+            return conn.get("region")
+    except Exception:
+        pass
```

Ruff (0.15.9)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)